runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -14,6 +14,9 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'lombok.Setter'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)